skins/cat17/src/app/components/BankData.vue

```diff
@@ -100,20 +109,14 @@
 			},
 			classesIBAN() {
 				return {
-					'field-grp': true,
-					'field-iban': true,
-					'field-labeled': true,
-					invalid: !this.isValid,
-					valid: this.isValid && !this.isAccountIdEmpty()
+					invalid: this.isValid === false,
```

Comment:
Because when `isValid` is null, that means that the validator does not have sufficcient data yet (e.g. classic bank data has only the account field filled). In those cases we don't want the fields become red immediately, because technically the data is not invalid, just incomplete. See also [ADR-04](https://github.com/wmde/FundraisingFrontend/blob/master/doc/adr/004_Client_Side_Validation.md)

Thanks for your question, it really validates Pablos comment at https://github.com/wmde/FundraisingFrontend/blob/master/skins/cat17/src/app/lib/reducers/validity.js#L4 
I might do a followup commit where I introduce the constants `VALID`/`INVALID`/`INCOMPLETE` to make it more obvious what is meant.